ng-dev/pr/merge/merge-pull-request.ts

```diff
@@ -134,3 +133,14 @@ async function createPullRequestMergeTool(flags: PullRequestMergeFlags) {
     throw e;
   }
 }
+
+/**
+ * Parses the pull request number from either the number or url string
+ */
+export function parsePrNumber(prUrlOrNumber: string): number {
+  const prNumber = parseInt(prUrlOrNumber.split('/').pop()!);
+  if (isNaN(prNumber)) {
+    throw new FatalMergeToolError('Pull Request was unable to be parsed from the parameters');
```

Comment:
Since this failure is now moved into the yargs parsing, we aren't running within the context of the merge tool, so `FatalMergeToolError` is not necessary, we should just through a generic error.